Apps/frontend/cypress/e2e/2-control-panel/reset-button.js

```diff
@@ -0,0 +1,17 @@
+/// <reference types="cypress" />
+
+describe("everything visible on front page", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:5173/");
+  });
+
+  it("displays btn", () => {
+    cy.get(".control-panel").should("be.visible");
+    cy.get(".btn-reset").should("be.visible");
```

Comment:
This is probably more for refactoring, but we currently use very speaking variable/function names. If you don't have any hard feelings I'd suggest to write out button (also in other files).

Reply:
Will be renamed.